packages/cuba-react-core/src/data/Collection.tsx

```diff
@@ -23,13 +24,16 @@ export class DataCollectionStore<T> implements DataContainer {
   @observable count?: number;
   @observable skipCount?: boolean;
 
+  mode?: 'server' | 'client';
```

Comment:
Not sure that I understand client\server mode. May be add comment here - when we use which mode.

Reply:
done